pkg/api/addons.go

```diff
@@ -29,17 +29,20 @@ func (cs *ContainerService) setAddonsConfig(isUpgrade bool) {
 	}
 	k8sComponents := GetK8sComponentsByVersionMap(o.KubernetesConfig)[o.OrchestratorVersion]
 	omsagentImage := "mcr.microsoft.com/azuremonitor/containerinsights/ciprod:ciprod05262020"
+	omsagentWinImage := "mcr.microsoft.com/azuremonitor/containerinsights/ciprod:win-ciprod05262020-2"
```

Comment:
Kaveesh, this is not related to your change but I am seeing clusterName tag not being created as part of the cluster creation and this should be there.  Lets look at your aks-engine cluster and validate this working. I would like to get this fix as part of your PR if feasible.

Reply:
I think we should create a separate PR for it since its not realted to this change + it will be easier to track it too.

I like that the PR's that get merged into this repository have 'feat: created new capability..', 'fix: some bug fix message..' etc. in their names so its easy to look at the history to find out what changes are being made.

Reply:
> I think we should create a separate PR for it since its not realted to this change + it will be easier to track it too.
> 
> I like that the PR's that get merged into this repository have 'feat: created new capability..', 'fix: some bug fix message..' etc. in their names so its easy to look at the history to find out what changes are being made.

> Ok, go-ahead and we can track that separate issue. Looks like someone broke that.